clustermesh-apiserver/clustermesh/cells.go

```diff
@@ -7,54 +7,28 @@ import (
 	"github.com/cilium/hive/cell"
 
 	cmk8s "github.com/cilium/cilium/clustermesh-apiserver/clustermesh/k8s"
-	"github.com/cilium/cilium/clustermesh-apiserver/health"
-	cmmetrics "github.com/cilium/cilium/clustermesh-apiserver/metrics"
-	"github.com/cilium/cilium/clustermesh-apiserver/option"
-	"github.com/cilium/cilium/clustermesh-apiserver/syncstate"
 	"github.com/cilium/cilium/pkg/clustermesh/operator"
 	cmtypes "github.com/cilium/cilium/pkg/clustermesh/types"
-	"github.com/cilium/cilium/pkg/controller"
-	"github.com/cilium/cilium/pkg/defaults"
-	"github.com/cilium/cilium/pkg/gops"
 	k8sClient "github.com/cilium/cilium/pkg/k8s/client"
 	"github.com/cilium/cilium/pkg/k8s/synced"
-	"github.com/cilium/cilium/pkg/kvstore"
 	"github.com/cilium/cilium/pkg/kvstore/heartbeat"
-	"github.com/cilium/cilium/pkg/kvstore/store"
-	"github.com/cilium/cilium/pkg/pprof"
 )
 
 var Cell = cell.Module(
 	"clustermesh",
 	"Cilium ClusterMesh",
 
-	cell.Config(option.DefaultLegacyClusterMeshConfig),
 	cell.Config(operator.MCSAPIConfig{}),
 
 	// We don't validate that the ClusterID is different from 0 (and the
 	// ClusterName is not the default one), because they are valid in
 	// case we only use the external workloads feature, and not clustermesh.
-	cell.Config(cmtypes.DefaultClusterInfo),
 	cell.Invoke(cmtypes.ClusterInfo.InitClusterIDMax),
 	cell.Invoke(cmtypes.ClusterInfo.Validate),
```

Comment:
Just a note that this part will be uniformed with KVStoreMesh via https://github.com/cilium/cilium/pull/37418